docs/samples/custom/prebuilt-image/README.md

```diff
@@ -0,0 +1,61 @@
+# Predict on a InferenceService using a prebuilt image
+
+## Setup
+
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#deploy-kfserving).
+2. Your cluster's Istio Ingress gateway must be network accessible.
+
+## Create the InferenceService
+
+Apply the CRD
+
+```
+kubectl apply -f custom.yaml
+```
+
+Expected Output
+
+```
+inferenceservice.serving.kubeflow.org/custom-prebuilt-image
+```
+
+## Run a prediction
+
+This example uses the [codait/max-object-detector](https://github.com/IBM/MAX-Object-Detector) image. Since its REST interface is different than the [Tensorflow V1 HTTP API](https://www.tensorflow.org/tfx/serving/api_rest#predict_api) that KFServing expects we will need to bypass the inferenceservice and send our request directly to the predictor.
```

Comment:
Maybe add one line for what this model is expecting for the input/output and the point out the prediction endpoint for this model is `/model/predict`?